### DISCUSSION TITLE
New make

### DISCUSSION THREAD

**Discussion by vukasinterzic** [martinwoodward/octolamp#7] [Show and tell]
Hi, thanks for the files, it works and looks great. I used a different board so I had to desolder/cut pins from it to make it fit :) And I added a little zip tie for power cable security.

![IMG_7502](https://user-images.githubusercontent.com/37403397/215527747-8290c49a-8c41-4344-b6de-93521eff2867.jpg)
![IMG_7496](https://user-images.githubusercontent.com/37403397/215527731-5d2e7f1f-213d-4f57-ab41-089c9392906c.jpg)

**Comment by martinwoodward** [comment]
Ouch, sorry about that. Which board was it - a NodeMCU esp8266 with presoldered headers? (Like [this](https://a.co/d/aplp55s))

I've been wondering if I should provide a thicker base option in case people are using boards with pre-soldered headers, this is making me think yes. Also adding some small holes in case someone else wants to do that cable tie thing

Your build came out fanatically well though. Good job for persevering!
